go.mod

```diff
@@ -81,6 +81,14 @@ require (
 	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
 )
 
+require (
+	github.com/coreos/go-oidc v2.2.1+incompatible // indirect
```

Comment:
looks like you need to run `go mod tidy`

Reply:
done